scripts/src/client.ts

```diff
@@ -417,6 +438,24 @@ export class Client {
 		}
 	}
 
+	public async createIncomingTransferOrder(walletAddress: string, appId: string, title: string, description: string, memo: string): Promise<OpenOrder> {
+		try {
+			const res = await this.requests
+				.request(`/v2/transfers/incoming/orders`, {
+						wallet_address: walletAddress,
+						app_id: appId,
+						title,
+						description,
+						memo,
+				 })
+				.post<OpenOrder>();
```

Comment:
Doesn't this return a regular Order?

Reply:
Right, it was returning OpenOrder but then I changed it to create the order as pending. Done.